client/package.json

```diff
@@ -8,7 +8,7 @@
     "lint:fix": "yarn lint --fix",
     "start": "react-scripts start",
     "test": "is-ci \"test:ci\" \"test:watch\"",
-    "test:ci": "CI=true tsc && react-scripts test --maxWorkers 2 --env=jest-environment-jsdom-sixteen --coverage",
+    "test:ci": "CI=true tsc && react-scripts test --maxWorkers 4 --env=jest-environment-jsdom-sixteen --coverage",
```

Comment:
this command isn't getting used anymore as far as I can tell

Reply:
isn't it called when `yarn --cwd client test` is called? (see the line right above it)

Reply:
oh lol true 🤦